test/Spec.hs

```diff
@@ -116,7 +123,9 @@ spec = hspec $ do
           mempty,
           M.fromList [(["default", "x"], parse "hello")
                      ,(["foo", "bar", "baz"], parse "<apply-content/>")]) `shouldRender` "hello"
-
+    it "should allow overriden Html tags" $ do
+      ("<html><div></div></html>",
+       RenderContext mempty (Overrides mempty (HS.fromList ["div"])) (fills [("div", text "notadivatall")])) `shouldRenderCustom` "<html>not a div at all</html>"
```

Comment:
@dbp this is maybe a flaw in that should Render helper that strips out the spaces of   the rendered and the expected text